Libraries/Components/WebView/WebView.android.js

```diff
@@ -137,6 +137,16 @@ class WebView extends React.Component {
     scalesPageToFit: PropTypes.bool,
 
     /**
+     * Function that is invoked when a defined URL-scheme has been blocked.
+     */
+    onUrlSchemeBlocked: PropTypes.func,
+
+    /**
+     * An array defining blacklisted URL-schemes.
+     */
+    urlSchemeBlacklist: PropTypes.array,
```

Comment:
Could we call this something like `rejectedUrlSchemes` and `onUrlSchemeRejected`?